src/cpp/session/SessionConsoleProcessInfo.cpp

```diff
@@ -194,6 +195,10 @@ core::json::Object ConsoleProcessInfo::toJson() const
    result["shell_type"] = static_cast<int>(shellType_);
    result["channel_mode"] = static_cast<int>(channelMode_);
    result["channel_id"] = channelId_;
+   result["alt_buffer"] = altBufferActive_;
+   result["cwd"] = cwd_.absolutePath();
```

Comment:
Does this ever go to the client? The client doesn't actually know what the user's home directory is so we usually convert to/from aliased paths when serializing/deserializing json containing path objects.